front/src/modules/ui/view-bar/components/MultipleFiltersDropdownButton.tsx

```diff
@@ -13,12 +14,21 @@ type MultipleFiltersDropdownButtonProps = {
 export const MultipleFiltersDropdownButton = ({
   hotkeyScope,
 }: MultipleFiltersDropdownButtonProps) => {
+  const { closeDropdownButton } = useDropdownButton({
+    dropdownId: FilterDropdownId,
+  });
+
+  const handleDropdownButtonClose = () => {
+    closeDropdownButton();
+  };
+
   return (
     <DropdownButton
       dropdownId={FilterDropdownId}
       buttonComponents={<MultipleFiltersButton />}
       dropdownComponents={<MultipleFiltersDropdownContent />}
       dropdownHotkeyScope={hotkeyScope}
+      onClose={handleDropdownButtonClose}
```

Comment:
let's let it to DropdownButton itself to close itself. The onClose callback will be used to trigger extra onClose behaviors such as resetState